File: hmw2/codes/bozkurt_alper.rb
#conditional exit
puts "..First Day.."
bragging = "I've read book"
for $page in 1..4 
  puts bragging 
end
puts  "It's enough for today."

#unconditional exit 
sleeping = 2
puts "..Second Day.."
for $page in 1..4
  puts bragging
  break if sleeping == $page
end
puts  "Dreams."

File: hmw2/codes/bozkurt_alper.rs
fn main() {

  //conditional exit
  let mut sleep = 1;
  let wake_up = 9;
  while sleep < wake_up{
      sleep += 1;
      println!("Dreams..");
  }
  println!("Run!");
  
  //unconditional exit
  sleep = 1;
  let neighbour_on = 3;
  while sleep < wake_up{
      if neighbour_on == sleep{ break; }
      sleep += 1;
      println!("Dreams..");
  }
  println!("Waking up in the middle of the night");
}